### DISCUSSION TITLE
Question: How to add a regular website

### DISCUSSION THREAD

**Discussion by tott** [linuxserver/Heimdall#1102] [Q&A] [answered]
I really love heimdall for managing my apps but I'd like it to be my general start page and also include my frequently visited websites that are not apps. However, somehow I could not find a "universal" application that would just grab the favicon / color or something similar and let me add general URLs to the dashboard. 

Am I missing something here? How does everyone do such things?

**Comment by keriati** [comment, marked as answer]
When you are adding an item just click on the Website button, enter the url and it should provide some icons and fill in the url:

![Screenshot 2023-01-18 at 21 24 34](https://user-images.githubusercontent.com/439392/213287018-1bb0aa3d-cc56-485d-b00b-0dca4e15febe.png)

**Comment by tott** [comment]
Thanks a lot - Seems I was just on an older version. After upgrading to the latest development image everything is working as outlined above. This is great.
